packages/create-hint/src/templates/locales-messages.json.hbs

```diff
@@ -0,0 +1,12 @@
+{
+    {{#each hints}}
+    "{{#if ../isMulti}}{{toCamelCase normalizedName}}_{{/if}}description": {
+        "description": "Metadata description",
+        "message": "{{description}}"
+    },
+    "{{#if ../isMulti}}{{toCamelCase normalizedName}}_{{/if}}name": {
+        "description": "Metadata name",
+        "message": "{{normalizedName}}"
```

Comment:
@sarvaje key name is in `camelCase`. But actual name value seems to be `space` delimited with first char capital. Right?
Do we have helper for that in utils? For me it seems like `normalize-string-with-delimiter` will work... But should I create one for same?

Reply:
can you give me an example?

Reply:
Like in `hint-axe` locale file...
https://github.com/webhintio/hint/blob/master/packages/hint-axe/src/_locales/en/messages.json

see `nameRoleValue_name` > `message` value
```
"nameRoleValue_name": {
        "description": "Metadata name",
        "message": "Name role value"
 },
```

I just see similar pattern in many hints though it is just a value, user can configure it.
I was asking for default value when hint gets created.

Reply:
Ahhh, yes, the actual name is like you said. I was talking only about the key. Sorry for the misunderstanding

Reply:
ok that is fine. So my changes are done. Please have a review once.